lib/_http_outgoing.js

```diff
@@ -78,11 +78,12 @@ exports.OutgoingMessage = OutgoingMessage;
 OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
-  if (!this.socket) {
+
+  if (!this.socket)
```

Comment:
this is hazardous styling IMO, personally I'd prefer braces where the body of a conditional spans more than one line, even if it's effectively a one-liner, simply for clarity